include/api.php

```diff
@@ -1904,14 +1905,14 @@ function api_favorites_create_destroy($type){
 
 		$a = get_app();
 
-		if (api_user()===false) throw new ForbiddenException();
+		if (api_user()===false) throw new HTTP\ForbiddenException();
 
 		// for versioned api.
 		/// @TODO We need a better global soluton
 		$action_argv_id=2;
 		if ($a->argv[1]=="1.1") $action_argv_id=3;
```

Comment:
Standards: Please add braces to this conditional statement and spaces around the `==` and `=` operators.